lib/rss.js

```diff
@@ -0,0 +1,190 @@
+              {
+                'org:name': organization.name,
+              },
+              {
+                'org:slug': organization.slug,
+              },
+              {
+                'org:id': organization.id,
+              },
+              {
+                'property:path': stringPath,
+              },
+              {
+                'property:type': kind === 'N' ? 'addition' : 'deletion',
+              },
+              {
+                'property:display_name': user.display_name,
+              },
+              {
+                'property:id': user.id,
+              },
+              {
+                'property:github_account': user.github_account,
+              },
+            ],
+          })
+        }
+
+        if (itemPath[0] === 'leaders') {
+          return
+        }
+
+        if (finalProperty === 'completed_task_instance_count') {
+          title = `Completed Tasks Updated for ${organization.name}`
+        } else if (finalProperty === 'rank') {
+          title = `Organization Rank Updated for ${organization.name}`
+        } else {
+          title = `Updated "${finalProperty}" for ${organization.name}`
+        }
+
+        if (kind === 'A') {
+          return feedItems.push({
+            title,
+            date: dataUpdated,
+            custom_elements: [
+              {
+                'org:name': organization.name,
+              },
+              {
+                'org:slug': organization.slug,
+              },
+              {
+                'org:id': organization.id,
+              },
+              {
+                'property:path': stringPath,
+              },
+              {
+                'property:type': item.kind === 'N' ? 'addition' : 'deletion',
+              },
+              {
+                'property:item': item.rhs,
+              },
+            ],
+          })
+        }
+
+        return feedItems.push({
+          title,
+          description: `${
+            organization.name
+          } changed ${finalProperty} from ${lhs} to ${rhs}.`,
+          date: dataUpdated,
+          custom_elements: [
+            {
+              'org:name': organization.name,
+            },
+            {
+              'org:slug': organization.slug,
+            },
+            {
+              'org:id': organization.id,
+            },
+            {
+              'property:path': stringPath,
+            },
+            {
+              'property:type': 'change',
+            },
+            {
+              'property:old': lhs,
+            },
+            {
+              'property:new': rhs,
+            },
+          ],
+        })
+      })
+
+      const feed = new RSS({
+        title: 'Google Code-in Leaders',
+        description: 'A feed for Google Code-in updates',
+        site_url: process.env.URL || 'https://gci-leaders.netlify.com',
+        feed_url: `${process.env.URL ||
+          'https://gci-leaders.netlify.com'}/build.xml`,
```

Comment:
why `build.xml`?  It doesnt exist...?

Reply:
Good catch. Should be `feed.xml`. Fixed in latest commit.